http2_general/test_h2_hpack.py

```diff
@@ -651,26 +649,92 @@ class TestHpackUnknownMethod(TestHpackBase):
         """
     }
 
-    def test_unknown_method_dynamic_table(self):
+    @marks.Parameterize.expand(
+        [marks.Param(name="huffman", huffman=True), marks.Param(name="no_huffman", huffman=False)]
+    )
+    def test_unknown_method_dynamic_table(self, name, huffman):
         """
         Verifies correctness of processing unknown method that
         stored in dynamic table.
         """
         client = self.get_client("deproxy")
         server = self.get_server("deproxy")
-        method = "UPDATEREDIRECTREF"
+        methods = ["UPDATEREDIRECTREF", "PUTA", "GETA", "OPTIONA"]
+        count = 1
 
         self.start_all_services()
 
-        request = client.create_request(method=method, headers=[])
+        config = self.get_tempesta().config.defconfig
+        self.get_tempesta().config.defconfig = (
+            config + "frang_limits {http_strict_host_checking false; http_methods unknown;}\n"
+        )
+        self.get_tempesta().reload()
```

Comment:
It's not necessary. Just call `start_all_services` method after the config change and remove it before that.